pkg/version/version.go

```diff
@@ -92,5 +91,42 @@ func GetLatestVersion(uuid string, source string) (string, error) {
 		return "", err
 	}
 
-	return versionRsp["version"], nil
+	channel := parseChannel(Version)
+	if channel == "" {
+		channel = "version"
+	}
+
+	version, ok := versionRsp[channel]
+	if !ok {
+		return "", fmt.Errorf("Unsupported version channel: %s", channel)
```

Comment:
when i run `linkerd check`, i get:

```
linkerd-version: can determine the latest version..........................[FAIL] -- Unsupported version channel: git
```

expected ? does this require modifying how use the `git-sha`-style tags?